src/OnlineSales/Entities/Account.cs

```diff
@@ -52,4 +54,12 @@ public class Account : BaseEntity
     [Searchable]
     [Column(TypeName = "jsonb")]
     public string? Data { get; set; }
+
+    [Ignore]
+    [JsonIgnore]
```

Comment:
There are should be explicit difference between [Ignore] and [JsonIgnore] otherwise it would be hard to see behaviour of  those attributes. e.g. [Nest.Ignore] and [Json.Ignore]